### PR TITLE
doc: add minutes for 11th Oct 2022

### PR DESCRIPTION
**Pull request by gireeshpunathil**
Fixes: https://github.com/nodejs/diagnostics/issues/592

### CODE REVIEW

**Comment by gireeshpunathil** [comment]
@nodejs/diagnostics

File: wg-meetings/2022-10-11.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=o2tBFhRWO7g
+* **GitHub Issue**: https://github.com/nodejs/diagnostics/issues/592
+* **Minutes Google Doc**: https://docs.google.com/document/d/1utEDIAUXoLFCPdqorSLacHlpdo5j4F20tDwf2Kum938/edit
+
+## Present
+
+* Diagnostics team: @nodejs/diagnostics
+* Stephen Belanger@qard
```
Comment (anonrig) [inline wg-meetings/2022-10-11.md:12]: Can you fix this spacing issue?

**Review by tony-go — APPROVED** [review]
+1 for @anonrig comment otherwise lgtm 🚀

**Review by RafaelGSS — APPROVED** [review]

File: wg-meetings/2022-10-11.md (hunk 1 above)
Comment (gireeshpunathil) [reply wg-meetings/2022-10-11.md]: done, thanks.

**Comment by gireeshpunathil** [comment]
landed in b7f281de7b33e032d802c468664cac725a94c3b4

**Pull request closed without merging**